coalib/bearlib/aspects/Smell.py

```diff
@@ -101,13 +100,32 @@ def func(a, b, c, d, e, f, g, h, i, j, k, l, m, n, o, p, q, r, s, t, z):
         parameters (maybe by creating more functions or using libraries).
         """
 
+
+@MethodSmell.subaspect
+class MethodLength:
+    """
+    This aspect checks for methods and functions with a large number
+    of lines.
+    """
+    class docs:
+        example = """
```

Comment:
This example is not a good example of what the aspect is responsible for. Maybe a 5 line method for something you can do in one or two lines very obviously. Like expand an inline if else clause to 4 lines.